src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -1287,7 +1287,11 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
                         $actionObject->getActionOrigin()
                     )[0];
                     $argRef = "\t\t\$";
-                    $argRef .= str_replace(ucfirst($fieldKey), "", $stepKey) . "Fields['{$fieldKey}'] = ${input};\n";
+
+                    $input = $this->resolveAllRuntimeReferences([$input])[0];
+                    $argRef .= str_replace(ucfirst($fieldKey), "", $stepKey) .
+                        "Fields['{$fieldKey}'] = ${input};\n";
```

Comment:
remove `\n` from end of string, do the same for the `magentoCLI` function when you touch that part of the codebase

Reply:
Updated.